From 28a1c1a88919cc4e69ea64e79097668a909d48a6 Mon Sep 17 00:00:00 2001
From: Morten Linderud <morten@linderud.pw>
Date: Sun, 13 Mar 2022 12:36:50 +0100
Subject: [PATCH] init/meson: Use libdir instead of hardcoded /lib path

Hardcoding `/lib` makes meson create a directory which would conflict on
distros with usrmerge as `/lib` is a symlink. We define `libdir` in the
top-level so we should be using that instead.

Signed-off-by: Morten Linderud <morten@linderud.pw>
---
 config/init/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/init/meson.build b/config/init/meson.build
index 5061973c..38b6a923 100644
--- a/config/init/meson.build
+++ b/config/init/meson.build
@@ -12,7 +12,7 @@ if init_script == 'systemd'
             '@OUTPUT@',
         ],
         install: true,
-        install_dir: '/lib/systemd/system')
+        install_dir: join_paths(libdir, 'systemd/system'))
 
 elif init_script == 'upstart'
         install_data('upstart/lxcfs.conf', install_dir: join_paths(sysconfdir, 'init'))
